src/style.css

```diff
@@ -0,0 +1,123 @@
+body {
+  font-family: 'Roboto', sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+main {
+  margin: 10% auto;
+  padding: 20px;
+}
+
+h1,
+h2 {
+  font-family: 'Roboto', sans-serif;
+  font-weight: 900;
+  margin: 0;
+}
+
+h1 {
```

Comment:
- [ ] Please make sure to fix the below style issues to match the wireframe design.
  - Please make sure the Leaderboard heading is aligned at the top of the scores section on the left. ( On big screens it's displaying on the left side, It should be visible at the top of the scores section on every screen).

### Your design

![image](https://github.com/lincoln1883/leaderboard/assets/83803414/951eadae-5151-4530-93f1-c17ec915d79f)

### Wireframe design

![image](https://github.com/vangelif/LB/assets/83803414/3888e615-7a91-492b-bb9d-2a4dcec754bd)

Reply:
I don't mean to kill your joy but your wide-screen monitor is over-exaggerating! however ill make the necessary changes thhank you.